lib/tasks/ahoy.rake

```diff
@@ -0,0 +1,14 @@
+namespace :ahoy do
+  desc "Filling empty geo-location entries in ahoy_visit table"
+
+  task fill_empty_location_entries: :environment do
```

Comment:
@lascau  ⚠️ this is not running for all subdomains. We need to make sure that it runs on every schema. see here for example: https://github.com/restarone/violet_rails/blob/master/lib/tasks/report.rake#L6

Reply:
your code is incorrect. Did you run it before committing?

Reply:
I updated the documentation to add a section about rake tasks: https://github.com/restarone/violet_rails/wiki/Getting-started-(development-cheatsheet)#rake-tasks

Reply:
thanks, did the modifications.